### PR TITLE
Move 1.8 tests to 2.0 tests 

### PR DESCRIPTION
**Pull request by GaoWei8**
## 迁移的脚本 
- logical:  logical_and, logical_or
- logical_not
-  reduce:  reduce_mean, reduce_sum, reduce_prod
- scale
- shape 
- softmax_with_cross_entropy 
- softsign
- squeeze

### CODE REVIEW

File: api/tests_v2/reduce.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+from common_import import *
+
+
+class ReduceConfig(APIConfig):
+    def __init__(self):
+        super(ReduceConfig, self).__init__('reduce')
+        self.feed_spec = {"range": [-1, 1]}
+        self.api_name = 'reduce_mean'
+        self.api_list = {
+            'reduce_mean': 'reduce_mean',
```
Comment (Xreki) [inline api/tests_v2/reduce.py:24]: reduce_xxx接口在2.0 API里面已经改名了：
- reduce_sum -> sum
- reduce_prod -> prod
- reduce_mean -> mean

现在alias建的太多了，导致老的API也能用。

File: api/tests_v2/scale.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#   Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
```
Comment (Xreki) [inline api/tests_v2/scale.py:1]: 2019 -> 2020

File: api/tests_v2/reduce.py (hunk 1 above)
Comment (GaoWei8) [reply api/tests_v2/reduce.py]: 已修改为新接口。

File: api/tests_v2/scale.py (hunk 2 above)
Comment (GaoWei8) [reply api/tests_v2/scale.py]: Done.

**Review by Xreki — APPROVED** [review]

**Pull request merged**